src/Integration/LocalServices/SolutionBindingInformationProvider.cs

```diff
@@ -31,13 +31,46 @@ public SolutionBindingInformationProvider(IServiceProvider serviceProvider)
         }
 
         #region ISolutionBindingInformationProvider
+        public bool IsSolutionBound()
+        {
+            return this.GetSolutionBinding() != null;
+        }
+
         public IEnumerable<Project> GetBoundProjects()
         {
+            return this.GetBoundProjects(this.GetSolutionBinding());
+        }
+
+        public IEnumerable<Project> GetUnboundProjects()
+        {
+            return this.GetUnBoundProject(this.GetSolutionBinding());
```

Comment:
No, I mean why can't the body of `private IEnumerable<Project> GetUnBoundProject(BoundSonarQubeProject binding)` be inlined into `public IEnumerable<Project> GetUnboundProjects()`?
And similarly for `private IEnumerable<Project> GetBoundProjects(BoundSonarQubeProject binding)` and `public IEnumerable<Project> GetBoundProjects()`.